staging/src/k8s.io/legacy-cloud-providers/azure/azure_routes.go

```diff
@@ -111,6 +111,7 @@ func (d *delayedRouteUpdater) updateRoutes() {
 
 	// No need to do any updating.
 	if len(d.routesToUpdate) == 0 {
+		klog.V(4).Info("updateRoutes: nothing to update, returning")
```

Comment:
This seems like it could be too verbose even for v=4 but personally don't know how often this check would run

Reply:
increased to 6